superset-frontend/package.json

```diff
@@ -92,10 +92,11 @@
     "@superset-ui/plugin-chart-echarts": "^0.15.17",
     "@superset-ui/plugin-chart-table": "^0.15.17",
     "@superset-ui/plugin-chart-word-cloud": "^0.15.17",
+    "@superset-ui/plugin-filter-antd": "^0.15.17",
     "@superset-ui/preset-chart-xy": "^0.15.17",
     "@vx/responsive": "^0.0.195",
     "abortcontroller-polyfill": "^1.1.9",
-    "antd": "^4.8.2",
+    "antd": "^4.9.3",
```

Comment:
Maybe now's the time to go for 4.9.4? It has a Tree bugfix that may become relevant here.

Reply:
I bumped it in the latest master merge